### PR TITLE
buildroot 2017.11, linux 4.9.64

### PR DESCRIPTION
**Pull request by ijl**
I could not get the ISO build to work, so this fixes it by upgrading the builder image and buildroot. There are new versions of glibc and so on, and the kernel is the latest patch in the 4.9 series. Nothing interesting in logs and works as expected for my use cases.

### CODE REVIEW

**Comment by aaron-prindle** [comment]
@minikube-bot test this please

**Comment by ijl** [comment]
Is the cross build only applying the last commit to master? I see `Commit message: "Merge 563b4bed5073da3c2ea0430bbb3e96e36643bf59 into aa7ddbc947ef254da93fb60d2671a491f70aad04"` followed by `*** BR2_GLOBAL_PATCH_DIR contains nonexistent directory /mnt/deploy/patch.  Stop.` (which is from 94629af).

**Comment by ijl** [comment]
It doesn't have /mnt, nevermind.

**Comment by ijl** [comment]
Updated to relative path.

**Comment by r2d4** [comment]
@minikube-bot ok to test

File: deploy/iso/minikube-iso/configs/minikube_defconfig [hunk 1]
```diff
@@ -1,54 +1,59 @@
...
 BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/rootfs-overlay"
 BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.9.64"
 BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
 BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/linux-4.9_defconfig"
+BR2_PACKAGE_GZIP=y
 BR2_PACKAGE_XZ=y
-BR2_PACKAGE_E2FSPROGS=y
+BR2_PACKAGE_FINDUTILS=y
```
Comment (dlorenc) [inline deploy/iso/minikube-iso/configs/minikube_defconfig]: Looks like we dropped a few packages here:

```shell
BR2_PACKAGE_E2FSPROGS=y
BR2_PACKAGE_OPENVMTOOLS10=y
BR2_PACKAGE_OPENVMTOOLS10_PROCPS=y
BR2_TARGET_TZ_INFO=y		
BR2_TARGET_TZ_ZONELIST="default"		
BR2_TARGET_LOCALTIME="Etc/UTC"
```

are all missing but were included before.

Comment (ijl) [reply deploy/iso/minikube-iso/configs/minikube_defconfig]: Everything but OPENVMTOOLS10 is in the generated `.config`.

The forked openvmtools package appears to be obsolete now, with buildroot at 10.1.0. I'll look at building with theirs.

**Comment by ijl** [comment]
@dlorenc see latest. This is using 2017.11, and so patches for flex are unnecessary. Here's the generated buildroot .config: https://gist.githubusercontent.com/ijl/3cd8bf0b5cc0857f09530f714537c143/raw/4beca2a51a5f11010d10f28a07045c73da16983c/.config

**Comment by ijl** [comment]
@dlorenc @r2d4 any thoughts? I see the image was built ok. The test failures look like a variety of environmental issues unrelated to the ISO itself to me, though I'm not exactly familiar.

**Comment by dlorenc** [comment]
Thanks!

**Pull request merged**